### DISCUSSION TITLE
How to see which issues are fixed in VS Code Insiders?

### DISCUSSION THREAD

**Discussion by PEZ** [microsoft/vscode-discussions#67] [Extension Development QnA] [answered]
Hi! I tracked down a [bug reported on Calva](https://github.com/BetterThanTomorrow/calva/issues/1801) to what seems to be a regression in VS Code. I haven't checked in which version of VS Code the regression happened, but notice that I can't reproduce it in VS Code Insiders.

In the Insiders Release Notes for version 1.70 I find a link to issues for the July 2022 milestone:

https://github.com/Microsoft/vscode/issues?q=is%3Aissue+milestone%3A%22July+2022%22

It's 20 pages. Trying to narrow down my search i fail to find anything relevant.

The bug I have in the latest released VS Code (Version: 1.69.1) is that the selection is not maintained when applying a `vscode.TextEdit.replace(range, newText)` edit using the full range of the document. Instead the cursor is placed at the end of the document after the edit is applied. This is fixed in 1.70, afaict, and I would like to reference the issue or commits, but can't find either.

Anyone has an idea about how to search for this? Adding `workspaceedit` to the search doesn't do it. Though searching the full VS Code issues list for `workspaceedit` does find me an open and recent issue about a failing integration test:

* [vscode#155238](https://github.com/microsoft/vscode/issues/155238)

Not much detail there, and I doubt it is related to the issue I am looking for.

**Comment by usernamehw** [comment, marked as answer]
In GitHub you can try to use a more specific label: `label:insiders-released`.

Or searching from google `vscode edit site:github.com` limited to "Past month".

Doesn't produce good results this time though.

Also searching `workspaceEdit` in the entire repo: the most TOP-LEFT input in the GitHub header (In this repository) - should also include Pull Requests, Commits ...

<img width="500" src="https://user-images.githubusercontent.com/9638156/179361109-b8fea748-0b4d-4e57-ae14-5045d42a3369.png">
